src/components/Sum.jsx

```diff
@@ -1 +1,24 @@
-// export function Sum() {}
+import React from 'react';
+
+function Sum({ a, b }) {
```

Comment:
I think it's better to use arrow function, just like in App component.
You can also add default values for a and b